src/Data/Managers/V2/Environment/GetEnvironmentSettings/useGetEnvironmentSettings.ts

```diff
@@ -16,13 +16,13 @@ interface GetEnvironmentSettings {
  * @returns {UseQueryResult<EnvironmentSettings, Error>} returns.useOneTime - Fetch environment settings with a single query.
  */
 export const useGetEnvironmentSettings = (): GetEnvironmentSettings => {
-  const get = useGetWithoutEnv()<{ data: EnvironmentSettings }>;
+  const get = useGet()<{ data: EnvironmentSettings }>;
 
   return {
-    useOneTime: (envId: string): UseQueryResult<EnvironmentSettings, Error> =>
+    useOneTime: (): UseQueryResult<EnvironmentSettings, Error> =>
       useQuery({
-        queryKey: ["get_environment_settings-one_time", envId],
-        queryFn: () => get(`/api/v2/environment_settings/${envId}`),
+        queryKey: ["get_environment_settings-one_time"],
+        queryFn: () => get("/api/v2/environment_settings"),
```

Comment:
isn't the `envId` needed?

Reply:
for this call the env is assigned in useGet